README.md

```diff
@@ -827,6 +827,70 @@ By default you should at least use the following files:
 * `proguard-okio.pro`
 * `proguard-gson.pro`
 
+## Unit testing with JUnit 4 or JUnit 5
```

Comment:
I suggest we use a level 3 header and nest it under "FAQ" above.

Reply:
I just updated the PR with your suggestion 👍